packages/components/src/sender/index.less

```
@@ -295,6 +295,28 @@
+    display: flex;
+    align-items: center;
+    width: max-content;
+    height: var(--tr-sender-input-height);
+    line-height: var(--tr-sender-input-line-height);
+    white-space: nowrap;
+    pointer-events: none;
+    will-change: transform;
+  }
+
+  &__input-field-wrapper.has-input-prefix {
+    :deep(.tiny-textarea__inner) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)